website/docs/events/m3-slides/.slides/os-materials.md

```diff
@@ -0,0 +1,24 @@
+---
+
+## OS materials
+
+---
+
+### Lecture
+
+---
+
+### Practical Sessions
+
+---
+
+### Quizzes
+
+---
+
+### Demos
+
+---
+
+### Homework
```

Comment:
```suggestion
### Homework Assignments
```

Reply:
These are just for orientation. The slide could even disappear in the final version.